### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by stevekuznetsov**
config: allow configuring kubeconfig path

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

server: allow absolute data directories

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

server: start controllers in their own goroutines

The code today blocks on the first Start() call, ensuring that the
second controller never starts and the server is never ready.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

---

### CODE REVIEW

File: pkg/server/server.go [hunk 1]
```diff
@@ -88,13 +87,18 @@ func (s *Server) Run(ctx context.Context) error {
...
 	}
 
-	dir := filepath.Join(".", s.cfg.RootDirectory)
+	var dir string
+	if filepath.IsAbs(s.cfg.RootDirectory) {
+		dir = s.cfg.RootDirectory
+	} else {
+		dir = filepath.Join(".", s.cfg.RootDirectory)
+	}
 	if len(s.cfg.RootDirectory) != 0 {
```
Comment (ncdc) [inline pkg/server/server.go:96]: Should this be checking `dir` instead?

Comment (ncdc) [reply pkg/server/server.go:96]: If not, feel free to ignore 😄

Comment (stevekuznetsov) [reply pkg/server/server.go:96]: /shrug

I don't see a semantic difference between the two and was happy with the smaller diff. We default the value anyway so defending against `len() == 0` is on the silly side.

**Comment by stevekuznetsov** [comment]
@ncdc this is updated for your comments :)

**Review by ncdc — APPROVED** [review]

**Pull request merged**